Cargo.toml

```diff
@@ -72,6 +72,7 @@ x509-parser = "0.16.0"
 features = ["dispatch"]
 
 [patch.crates-io]
+cbor-smol = { git = "https://github.com/sosthene-nitrokey/cbor-smol.git", rev = "94ee8c28edf9248b402aa4335c1dee157995197b"}
```

Comment:
I fear that this is a change that could easily be missed when updating dependencies, breaking existing state.  What do you think about introducing a dummy feature in the fork like `bytes-from-array` so that we get an error if we don’t have the correctly patched `cbor-smol` version?

Reply:
Actually this change is already in `nitrokey-3-firmware`